lib/evmone/delegation.hpp

```diff
@@ -23,20 +23,23 @@ inline constexpr bool is_code_delegated(bytes_view code) noexcept
 }
 
 /// Get EIP-7702 delegate address from the code of addr, if it is delegated.
+/// TODO: This should not be inline function.
```

Comment:
@chfast should I move it to delegation.cpp?

Reply:
I think yes. You may also swap parameters, I think `host`, `addr` would be more natural. But up to you.